packages/node_modules/@node-red/runtime/lib/nodes/flows/Subflow.js

```diff
@@ -192,6 +223,42 @@ class Subflow extends Flow {
         super.start(diff);
     }
 
+    /**
+     * Get environment variable of subflow
+     * @param {String}   name   name of env var
+     * @return {Object}  val    value of env var
+     */
+    getSetting(name) {
+        var node = this.node;
+        if (node) {
+            var env = node.env;
+            if (env && env.hasOwnProperty(name)) {
+                return env[name].value;
+            }
+        }
+        var parent = this.parent;
+        if (parent) {
+            var val = parent.getSetting(name);
+            return val;
+        }
+        return undefined;
+    }
+
+    /**
+     * Set environment variable of subflow
+     * @param {String}   name   name of env var
+     * @param {Object}   val    value of env var
+     */
+    setSetting(name, val) {
```

Comment:
There's no current requirement to have `setSetting` - the settings are read-only. We can revisit that in the future, but I don't want to add more code than strictly needed. If a node author discovered this api and started using it we'd be stuck with it and there may be unintended consequences.

Reply:
This was added for use in test cases.  But removed from updated code.